cd/cd_dump.ixx

```diff
@@ -712,7 +712,7 @@ export bool redumper_dump_cd(Context &ctx, const Options &options, bool refine)
             // PLEXTOR: multisession lead-out overread
             // usually there are couple of slow sectors before SCSI error is generated
             // some models (PX-708UF) exit on I/O semaphore timeout on such slow sectors
-            if(ctx.drive_config.type == DriveConfig::Type::PLEXTOR && slow && inside_range(lba, error_ranges) != nullptr)
+            if((ctx.drive_config.type == DriveConfig::Type::PLEXTOR || ctx.drive_config.product_id == "CD-R PX-W4824A") && slow && inside_range(lba, error_ranges) != nullptr)
```

Comment:
This check has a tiny issue: if drive is "LITEON CD-R PX-W4824A" it will pass the check and we don't want that (in an unlikely scenario of vendor id being something else than PLEXTOR). 

I added a function to dump.ixx: drive_is_plextor4824() - please use it to check if it's 4824 drive - update the PR and I will approve it.